modules/oracledb/oracledb.go

```diff
@@ -0,0 +1,629 @@
+package oracledb
+
+import (
+	"database/sql"
+	"strconv"
+	"strings"
+
+	"github.com/netdata/go.d.plugin/modules"
+
+	"gopkg.in/goracle.v2"
+)
+
+const defaultDSN = "SYSTEM/Oracle12345@ORCL" // there is no such thing as default connstr/dsn, but...
```

Comment:
If there is no such thing as default dsn please don't set default. Just check in `Init` if it's set and if not return `false`.

Reply:
✅